examples/osgviewerIPhone/shaders.h

```diff
@@ -0,0 +1,113 @@
+#pragma once 
```

Comment:
Thanks for the revisions. End of day here so I'll have a look on Monday.

I have just checked in the GL3+GL2 shaders into osgText, to be found in src/osgText/Font.cpp.  I'm hoping these will work for GLES3+GLE2.  These are now in textrefactor_3 branch